src/keymap.cpp

```diff
@@ -415,7 +415,7 @@ static const std::vector<OpDesc> opdescs = {
 	},
 	{
 		OP_DOWNLOAD_FULL_PAGE,
-		"download-full-page",
+		"download-full-page-op",
```

Comment:
Why rename it? Are you trying to avoid confusion between the setting and the operation?

Reply:
@Minoru I fixed [this](https://github.com/newsboat/newsboat/pull/2609/checks?check_run_id=19716842793) asciidoctor warning at CI

Reply:
Oh, sorry, I should've looked at the individual commits to figure that out. This is a problem indeed. This also gives us another opportunity to think about the name of the operation.

I want to ask others for their opinion, so let me summarize the problems and our options (as I see them).

Problems:

1. we need a name for the operation that replaces current item's description with one fetched from the item's link. This is exactly what `download-full-page` setting does, but the operation can be invoked discreetly while the setting applies to all items in the feed;
2. we need to be able to link to both the setting and the operation in our docs. Currently, links are generated from the setting/operation name, so if we use the same name for a setting and an operation, they clash. The way we generate links can be changed, but that's a chunk of work, and it's be nice if we could avoid that work.

We should strive to solve these such that there's the least confusion for the user.

I think there are three options:

1. call both the setting and the op `download-full-page`;
4. call the op `download-full-page-op` (or use some other suffix/prefix);
5. call the op completely differently, e.g. `fetch-full-page`.

The last two options work okay with Asciidoc as-is. The first option requires changes to how we link to settings/ops.

The second option I find quite inelegant. Do others feel the same way?

Between the first and third options, I'm struggling to pick the best one. The first one creates confusion when we're talking about the feature. "Oh, just use `download-full-page`" -- do I mean the setting here, or the operation? The third option makes it harder to find out that the feature is available both as a setting and an option. Well, okay, that last bit can probably be ameliorated with cross-linking between docs for the setting and the op, but I'm not entirely convinced.

Anyone has an opinion on this?

Reply:
On IRC, \_nikko suggests `refresh-description`.

Reply:
@Minoru no `refresh-description` sounds like an absolutely different thing.

I suggest next variants

- `download-full-page-manually`
- `download-current-page`
- `download-specific-page`
etc.

Conversation on naming could be endless, cause it's very individual/subjective topic, just pick what's your like more and I will implement it.